cal/cal/migrations/0047_auto_20170220_0038.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Was adding this migration intentional? It works when I re-run migrate but just making sure